### PR TITLE
Add tooltips to context menus

### LINKED ISSUE #13243: Tab context menu entries should have tooltips
This is sometimes needed as a last resort to find some info printed to the screen but unsaved anywhere else. Currently for doing this I had to do manual selection with mouse and then wait for ~1min for it to select all of scrollable viewport (hoping that selection doesn't break since it's quite jumpy), then copy to Notepad, then search.

So either allowing to copy the scrollable viewport into buffer and/or searching it directly can be useful.

### PR DESCRIPTION
**Pull request by cinnamon-msft**
Add tooltips to the tab context menu and the tab dropdown menu.

Closes #13243

### CODE REVIEW

File: src/cascadia/TerminalApp/TerminalPage.cpp [hunk 1]
```diff
@@ -940,6 +944,10 @@ namespace winrt::TerminalApp::implementation
                 // Create the command palette button.
                 auto commandPaletteFlyout = WUX::Controls::MenuFlyoutItem{};
                 commandPaletteFlyout.Text(RS_(L"CommandPaletteMenuItem"));
+                auto commandPaletteToolTip = RS_(L"CommandPaletteToolTip");
```
Comment (carlos-zamora) [inline src/cascadia/TerminalApp/TerminalPage.cpp:947]:
```suggestion
                const auto commandPaletteToolTip = RS_(L"CommandPaletteToolTip");
```

File: src/cascadia/TerminalApp/TerminalPage.cpp [hunk 2]
```diff
@@ -923,6 +923,10 @@ namespace winrt::TerminalApp::implementation
                 // Create the settings button.
                 auto settingsItem = WUX::Controls::MenuFlyoutItem{};
                 settingsItem.Text(RS_(L"SettingsMenuItem"));
+                auto settingsToolTip = RS_(L"SettingsToolTip");
```
Comment (carlos-zamora) [inline src/cascadia/TerminalApp/TerminalPage.cpp:926]:
```suggestion
                const auto settingsToolTip = RS_(L"SettingsToolTip");
```

File: src/cascadia/TerminalApp/TabBase.cpp [hunk 3]
```diff
@@ -103,6 +111,10 @@ namespace winrt::TerminalApp::implementation
         });
         closeTabMenuItem.Text(RS_(L"TabClose"));
         closeTabMenuItem.Icon(closeSymbol);
+        auto closeTabToolTip = RS_(L"TabCloseToolTip");
```
Comment (carlos-zamora) [inline src/cascadia/TerminalApp/TabBase.cpp:114]:
```suggestion
        const auto closeTabToolTip = RS_(L"TabCloseToolTip");
```

File: src/cascadia/TerminalApp/TabBase.cpp [hunk 4]
```diff
@@ -88,6 +92,10 @@ namespace winrt::TerminalApp::implementation
             }
         });
         _closeOtherTabsMenuItem.Text(RS_(L"TabCloseOther"));
+        auto _closeOtherTabsToolTip = RS_(L"TabCloseOtherToolTip");
+
+        WUX::Controls::ToolTipService::SetToolTip(_closeOtherTabsMenuItem, box_value(_closeOtherTabsToolTip));
+        Automation::AutomationProperties::SetHelpText(_closeOtherTabsMenuItem, _closeOtherTabsToolTip);
```
Comment (carlos-zamora) [inline src/cascadia/TerminalApp/TabBase.cpp:98]:
```suggestion
        const auto closeOtherTabsToolTip = RS_(L"TabCloseOtherToolTip");

        WUX::Controls::ToolTipService::SetToolTip(_closeOtherTabsMenuItem, box_value(closeOtherTabsToolTip));
        Automation::AutomationProperties::SetHelpText(_closeOtherTabsMenuItem, closeOtherTabsToolTip);
```

File: src/cascadia/TerminalApp/TabBase.cpp [hunk 5]
```diff
@@ -79,6 +79,10 @@ namespace winrt::TerminalApp::implementation
             }
         });
         _closeTabsAfterMenuItem.Text(RS_(L"TabCloseAfter"));
+        auto _closeTabsAfterToolTip = RS_(L"TabCloseAfterToolTip");
+
+        WUX::Controls::ToolTipService::SetToolTip(_closeTabsAfterMenuItem, box_value(_closeTabsAfterToolTip));
+        Automation::AutomationProperties::SetHelpText(_closeTabsAfterMenuItem, _closeTabsAfterToolTip);
```
Comment (carlos-zamora) [inline src/cascadia/TerminalApp/TabBase.cpp:85]:
```suggestion
        const auto closeTabsAfterToolTip = RS_(L"TabCloseAfterToolTip");

        WUX::Controls::ToolTipService::SetToolTip(_closeTabsAfterMenuItem, box_value(closeTabsAfterToolTip));
        Automation::AutomationProperties::SetHelpText(_closeTabsAfterMenuItem, closeTabsAfterToolTip);
```

**Review by carlos-zamora — APPROVED** [review]

File: src/cascadia/TerminalApp/Resources/en-US/Resources.resw [hunk 6]
```diff
@@ -746,4 +746,40 @@
...
+    <value>Open the search dialog</value>
+  </data>
+  <data name="RenameTabToolTip" xml:space="preserve">
+    <value>Rename this tab</value>
+  </data>
+  <data name="SettingsToolTip" xml:space="preserve">
+    <value>Open the settings page</value>
+  </data>
+  <data name="SplitTabToolTip" xml:space="preserve">
+    <value>Open a new pane using the active profile in the current directory</value>
```
Comment (DHowett) [inline src/cascadia/TerminalApp/Resources/en-US/Resources.resw:774]: (technically) :/

**Review by DHowett — APPROVED** [review]

**Comment by DHowett** [comment]
my only contention: "open the about dialog" means as much as "About" does, but with more words.
Should this instead say something like... "Open a dialog containing product information" or similar?

**Comment by cinnamon-msft** [comment]
> 

Updated :)

**Pull request merged**